src/examples/test-and-release-package-with-merge.yml

```diff
@@ -24,6 +24,11 @@ usage:
             upload-coverage: true
         - release-management/test-package:
             name: node-12
+            node_version: '12.16.3'
```

Comment:
Can you just say `12`?

Reply:
@amphro you can for linux but not windows because of how `choco` works... I can probably add some code to do that logic for us if you think it's important enough